pkg/rhos/ocpgwdeployer.go

```diff
@@ -157,14 +157,22 @@ func (d *ocpGatewayDeployer) Deploy(input api.GatewayDeployInput, status reporte
 		return status.Error(err, "creating gateway security group failed")
 	}
 
-	status.Success("Opened External ports %q in security group %q on RHOS",
-		formatPorts(input.PublicPorts), groupName)
-
 	gwNodes, err := d.K8sClient.ListGatewayNodes()
 	if err != nil {
 		return status.Error(err, "listing the existing gateway nodes failed")
 	}
 
+	gwNodesList := gwNodes.Items
+	for i := range gwNodesList {
+		err := d.openGatewayPort(groupName, gwNodesList[i].Name, computeClient)
+		if err != nil {
+			return status.Error(err, "failed to open the g/w pods in the existing g/w node")
```

Comment:
```suggestion
			return status.Error(err, "failed to open the g/w port in the existing g/w node")
```

and add the failing entry to the error message if appropriate.